docs/modules/ROOT/partials/getting-started/con-registry-serdes-constants.adoc

```diff
@@ -2,73 +2,82 @@
 [discrete]
-== Constants for serializer/deserializer services
+== Configuration for SerDe services
 
 [source,java,subs="+quotes,attributes"]
 ----
-public abstract class AbstractKafkaSerDe<T extends AbstractKafkaSerDe<T>> implements AutoCloseable {
-   protected final Logger log = LoggerFactory.getLogger(getClass());
+public class SerdeConfig {
 
-   public static final String REGISTRY_URL_CONFIG_PARAM = "apicurio.registry.url"; <1>
-   public static final String REGISTRY_CACHED_CONFIG_PARAM = "apicurio.registry.cached"; <2>
-   public static final String REGISTRY_ID_HANDLER_CONFIG_PARAM = "apicurio.registry.id-handler"; <3>
-   public static final String REGISTRY_CONFLUENT_ID_HANDLER_CONFIG_PARAM = "apicurio.registry.as-confluent"; <4>
+   public static final String REGISTRY_URL = "apicurio.registry.url"; <1>
+   public static final String ID_HANDLER = "apicurio.registry.id-handler"; <2>
+   public static final String REGISTRY_CONFLUENT_ID_HANDLER = "apicurio.registry.as-confluent"; <3>
 ----
-<1> (Required) The URL of {registry}.
-<2> Allows the client to make the request and look up the information from a cache of previous results, to improve processing time.
-If the cache is empty, the lookup is performed from {registry}.
-<3> Extends ID handling to support other ID formats and make them compatible with {registry} SerDe services.
-For example, changing the ID format from `Long` to `Integer` supports the Confluent ID format.
-<4> A flag to simplify the handling of Confluent IDs. If set to `true`, an `Integer` is used for the global ID lookup.
+<1> The required URL of {registry}.
+<2> Extends ID handling to support other ID formats and make them compatible with {registry} SerDe services.
+For example, changing the default ID format from `Long` to `Integer` supports the Confluent ID format.
+<3> Simplifies the handling of Confluent IDs. If set to `true`, an `Integer` is used for the global ID lookup.
+The setting should not be used with the `ID_HANDLER` option.
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more details, see the link:https://github.com/Apicurio/apicurio-registry/blob/master/serdes/serde-common/src/main/java/io/apicurio/registry/serde/SerdeConfig.java[SerdeConfig Java class] 
 
 [discrete]
-== Constants for lookup strategies
+== Configuration for SerDe lookup strategies
 
 [source,java,subs="+quotes,attributes"]
 ----
-public abstract class AbstractKafkaStrategyAwareSerDe<T, S extends AbstractKafkaStrategyAwareSerDe<T, S>> extends AbstractKafkaSerDe<S> {
-   public static final String REGISTRY_ARTIFACT_ID_STRATEGY_CONFIG_PARAM = "apicurio.registry.artifact-id"; <1>
-   public static final String REGISTRY_GLOBAL_ID_STRATEGY_CONFIG_PARAM = "apicurio.registry.global-id"; <2>
+public class SerdeConfig {
+    
+   public static final String ARTIFACT_RESOLVER_STRATEGY = "apicurio.registry.artifact-resolver-strategy"; <1>
+   public static final String REGISTRY_GLOBAL_ID_STRATEGY = "apicurio.registry.global-id"; <2>
```

Comment:
there is no global id strategy, this line and the referenced ones could be removed.
As I think I mentioned before, now we have `SchemaResolver` instead of global id strategy, but we want to encourage users to don't specify a Schema Resolver so the `DefaultSchemaResolver` will be used and then users will use the config properties that `DefaultSchemaResolver` accepts (see `io.apicurio.registry.serde.config.DefaultSchemaResolverConfig`) in order to do the things they did with the old `GlobalIdStrategy` and more